### ISSUE TITLE
Please add support for Emojis

### ISSUE THREAD

**Issue by approximate** [SuperChamp234/habitica-sync#10]
Thank you for the plugin, this is one webpage less to keep open =)

Currently any emojis that Habitica website and mobile client can display are represented as plain text in Habitica pane

![image](https://user-images.githubusercontent.com/1837916/141972088-614f4296-8416-429d-8dd0-513d20f391c0.png)

**Comment by SuperChamp234** [comment]
Looks like `:this:` type emoji's aren't getting rendered, gotta fix that.

Thanks for bringing this to our attention! We'll fix it asap!

![image](https://user-images.githubusercontent.com/40035606/141986401-52e1f3ad-1448-45e9-8067-61ca7d256b74.png)

**Comment by approximate** [comment]
@SuperChamp234 great work, thank you very much for such a quick turnaround!

I have tested the updated version, all emojis are now displayed correctly:
![image](https://user-images.githubusercontent.com/1837916/142403459-d057aa27-af93-43ff-ad01-eea507252c0a.png)

I have identified another markdown rendering problem, I will open another issue for it. Feel free to close this one, and thanks again for a quick fix!
